provisioning/src/EnvCollector.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Provisioning;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+final class EnvCollector
+{
+    private string $envFile;
+
+    public function __construct()
+    {
+        $this->envFile = __DIR__ . '/../env_export';
+    }
+
+    public function addEnv(string $name, string $value): void
+    {
+        $fs = new Filesystem();
+        if (!$fs->exists($this->envFile)) {
+            $fs->touch($this->envFile);
+        }
+        $fs->appendToFile($this->envFile, sprintf('export %s=%s' . PHP_EOL, $name, $value));
```

Comment:
Tady bys klidně mohl mít ještě druhý soubor, který by byl ve formátu v jakém se nastavují ty variables `              echo "##vso[task.setvariable variable=SYNAPSE_UID]$SYNAPSE_UID"` a ten si pak jen vyechovat místo vypisování přímo v pipelines.yml :)

Reply:
Dobrý nápad https://github.com/keboola/php-db-import-export/pull/39#discussion_r558273447 funguje to